### PR TITLE
Add audit section to the docs

### PR DESCRIPTION
**Pull request by thec00n**
List all audits that have been conducted for the plasma contracts and reference the reports.

### CODE REVIEW

File: plasma_framework/docs/audits/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Audits 
+
+The following list contains information on all the audits that have been conducted for the Plasma contracts component of the OMG network. 
+
+| Release | Organization | Period | Report |
+|--|--|--|--|
+| [v0.0.1](https://github.com/omisego/plasma-contracts/releases/tag/v0.0.1) | Quantstamp | 24.09.2018 - 10.11.2018 | [Report](https://github.com/omisego/plasma-contracts/blob/v0.0.1/docs/quantstamp-audit-3cc6097.md)|
+| [v1.0.0](https://github.com/omisego/plasma-contracts/releases/tag/v1.0.0) | Consensys Diligence| 21.10.2019 - 22.11.2019 and 16.12.2019 - 03.01.2020 | [Report](https://diligence.consensys.net/audits/2020/01/omisego-morevp/) |
```
Comment (boolafish) [inline plasma_framework/docs/audits/README.md]: have we considered to have a copy of pdf ourselves? I don't know, probably we can have link to pdf under this repo + the link to their page?

File: plasma_framework/docs/audits/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Audits 
+
+The following list contains information on all the audits that have been conducted for the Plasma contracts component of the OMG network. 
+
+| Release | Organization | Period | Report |
+|--|--|--|--|
+| [v0.0.1](https://github.com/omisego/plasma-contracts/releases/tag/v0.0.1) | Quantstamp | 24.09.2018 - 10.11.2018 | [Report](https://github.com/omisego/plasma-contracts/blob/v0.0.1/docs/quantstamp-audit-3cc6097.md)|
```
Comment (boolafish) [inline plasma_framework/docs/audits/README.md]: Do you mind if we use ISO date format instead? `yyyy-mm-dd` 

https://www.iso.org/iso-8601-date-and-time-format.html

Comment (thec00n) [reply plasma_framework/docs/audits/README.md]: ok

File: plasma_framework/docs/audits/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Audits 
+
+The following list contains information on all the audits that have been conducted for the Plasma contracts component of the OMG network. 
+
+| Release | Organization | Period | Report |
+|--|--|--|--|
+| [v0.0.1](https://github.com/omisego/plasma-contracts/releases/tag/v0.0.1) | Quantstamp | 2018.09.24 - 2018.11.10 | [MD](https://github.com/omisego/plasma-contracts/blob/v0.0.1/docs/quantstamp-audit-3cc6097.md)|
+| [v1.0.0](https://github.com/omisego/plasma-contracts/releases/tag/v1.0.0) | Consensys Diligence| 2019.10.21 - 2019.11.22 and 2019.12.16 - 2020.01.03 | [Link](https://diligence.consensys.net/audits/2020/01/omisego-morevp/) [PDF](./docs/Diligence_Morevp_Report.pdf)|
```
Comment (boolafish) [inline plasma_framework/docs/audits/README.md]:
```suggestion
| [v1.0.0](https://github.com/omisego/plasma-contracts/releases/tag/v1.0.0) | Consensys Diligence| 2019.10.21 - 2019.11.22 and 2019.12.16 - 2020.01.03 | [Link](https://diligence.consensys.net/audits/2020/01/omisego-morevp/), [PDF](./docs/Diligence_Morevp_Report.pdf)|
```

File: plasma_framework/docs/audits/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+## Audits 
+
+The following list contains information on all the audits that have been conducted for the Plasma contracts component of the OMG network. 
+
+| Release | Organization | Period | Report |
+|--|--|--|--|
+| [v0.0.1](https://github.com/omisego/plasma-contracts/releases/tag/v0.0.1) | Quantstamp | 2018.09.24 - 2018.11.10 | [MD](https://github.com/omisego/plasma-contracts/blob/v0.0.1/docs/quantstamp-audit-3cc6097.md)|
+| [v1.0.0](https://github.com/omisego/plasma-contracts/releases/tag/v1.0.0) | Consensys Diligence| 2019.10.21 - 2019.11.22 and 2019.12.16 - 2020.01.03 | [URL](https://diligence.consensys.net/audits/2020/01/omisego-morevp/) [PDF](./docs/Diligence_Morevp_Report.pdf)|
```
Comment (boolafish) [inline plasma_framework/docs/audits/README.md]:
```suggestion
| [v1.0.0](https://github.com/omisego/plasma-contracts/releases/tag/v1.0.0) | Consensys Diligence| 2019.10.21 - 2019.11.22 and 2019.12.16 - 2020.01.03 | [URL](https://diligence.consensys.net/audits/2020/01/omisego-morevp/), [PDF](./docs/Diligence_Morevp_Report.pdf)|
```

**Pull request merged**